.circleci/config.yml

```diff
@@ -6,12 +6,12 @@ jobs:
   build-and-test:
     working_directory: ~/project
     docker:
-      - image: cimg/node:18.18.0
+      - image: cimg/node:20.11.0
```

Comment:
CircleCI didn't yet have a container with 20.11.1 installed, but we can install it specifically below when we run the tests.